cfme/automate/explorer/method.py

```diff
@@ -228,6 +229,37 @@ def read(self):
         return self.all_vars
 
 
+class EntryPoint(Text, ClickableMixin):
+    def fill(self, value):
+        """Used to select the automate method from automate domain"""
+        if value:
+            self.click()
+            self.parent_view.embedded_view.wait_displayed('15s')
+            self.parent_view.embedded_view.tree.click_path(*value)
+            return True
+        return None
```

Comment:
The fill method should return boolean, and that boolean should represent whether or not a change was made on the view.

This needs to compare `value` to the currently selected path in the tree, clicking a new one and returning `True`, or just returning `False` if the currently selected path matches `value`